juriscraper/opinions/united_states/state/nev.py

```diff
@@ -35,13 +52,48 @@ def _download(self):
 
     def _process_html(self):
         for case in self.html:
-            if "COA" in case["caseNumber"]:
+            if not self.correct_court(case):
                 continue
+
+            if (self.now - datetime.fromisoformat(case["date"])).days > 30:
+                # Stop at 30 days
+                break
             self.cases.append(
                 {
                     "name": case["caseTitle"],
                     "docket": case["caseNumber"],
                     "date": case["date"],
-                    "url": case["docurl"],
+                    "url": "placeholder",
                 }
             )
+
+    def fetch_document_link(self, csNumber: str):
+        """"""
+        data = {
+            "action": "",
+            "csNumber": csNumber,
+            "shortTitle": "",
+            "exclude": "off",
+            "startRow": "1",
+            "displayRows": "50",
+            "orderBy": "CsNumber",
+            "orderDir": "DESC",
+            "href": "/public/caseView.do",
+            "submitValue": "Search",
+        }
+        content = self.request["session"].post(self.search, data=data).text
+        slug = fromstring(content).xpath(self.xp)[0]
```

Comment:
Maybe changing the index to `-1`, in case there are more than one opinion?
I haven't found any case with more than 1, though
```
        slug = fromstring(content).xpath(self.xp)[-1]
```